localstack/services/s3/s3_listener.py

```diff
@@ -241,21 +241,11 @@ def append_cors_headers(bucket_name, request_method, request_headers, response):
             for allowed in allowed_origins:
                 if origin in allowed or re.match(allowed.replace('*', '.*'), origin):
                     response.headers['Access-Control-Allow-Origin'] = origin
+                    if 'ExposeHeader' in rule:
+                        expose_headers = rule['ExposeHeader']
+                        response.headers['Access-Control-Expose-Headers'] = \
+                            ','.join(expose_headers) if isinstance(expose_headers, list) else expose_headers
                     break
-        # add additional headers
-        exposed_headers = rule.get('ExposeHeader', [])
-        for header in exposed_headers:
-            if header.lower() == 'date':
-                response.headers[header] = timestamp(format='%a, %d %b %Y %H:%M:%S +0000')
-            elif header.lower() == 'etag':
```

Comment:
I understand why we need to add the `Access-Control-Expose-Headers` header - that makes perfect sense (good catch!). 👍 

However, is it on purpose that we delete all these headers here? I'm guessing `ETag` and other headers are already exposed from the upstream response (which we receive from `moto`, the underlying S3 mock), right? @devoto13

Reply:
Thanks for the feedback, @whummer!

Yes, it is on purpose. We get the real values for some of these headers (`ETag`, `x-amz-delete-marker`, `x-amz-version-id`) from `moto` library, so not reason to set dummy values. 

Furthermore `ExposeHeader` property should only be used to set `Access-Control-Expose-Headers` header, which allows browser to read headers returned by the server, not add extra headers to the response.